content/el/template.json

```diff
@@ -3,10 +3,10 @@
   "logo-text":"io.js",
   "faq-link":"FAQ",
   "es6-link":"ES6",
-  "api-link":"API Docs",
+  "api-link":"API",
```

Comment:
Well, I was trying to fit something, but greek works are too damn long. They break the lines everywhere. Also the 'API' term is common, so writing it with a greek word for 'docs' next to it would not look so nice.
Same for FAQ, english term is common.
I'l see if anyone else on the team can come up with an idea.